tests/test_cmdline.py

```diff
@@ -40,9 +53,12 @@ def test_sig_stfl(sig_stfl_name):
             [helpers.path_to_executable('test_sig_stfl'), sig_stfl_name, katfile],
         )
     else:
-        helpers.run_subprocess(
-            [helpers.path_to_executable('test_sig_stfl'), sig_stfl_name],
-        )
+        if sig_stfl_name not in names_to_allow:
+            pytest.skip('Test skipped')
```

Comment:
Consider providing more details in the this message. For example, "skipping slow stateful sig tests; use command line directly if needed."